### DISCUSSION TITLE
Support `noscript` environments

### DISCUSSION THREAD

**Discussion by Empty2k12** [plausible/analytics#388] [Anything Else]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request
In environments where client-side JavaScript is not allowed, there could be a fallback using the `noscript` tag. Using this tag, an image could be added to the DOM when JavaScript is unavailable which can still be used to track users.

**How would you like it to work**
Ideally, next to the `plausible.js` there would be a route called `image.gif` which serves a `[1-9]x[1-9]` pixel image. Simple meta-data like page visit and user location can be detected from this. Metrics like page visit length cannot be tracked using this method, however having lower quality data is better than having no data.

**Comment by renbaoshuo** [comment]
Is there any progress on this feature request? I also encountered the same problem.

**Reply by metmarkosaric** [reply]
No progress on this. We're a small team with limited development resources so need to take it one step at a time. If you want to count those that don't use JavaScript, our events API is currently the best option. See https://plausible.io/docs/events-api

**Comment by yagarea** [comment]
Lack of this feature was the reason I stopped paying for plausible and started using goatcounter. But ui of gc is horrible. Implementation of this feature would easily make me go back to plausible.

I am sure I am the only one is not using plausible because of `no javascript` option.

**Reply by ytmauzer** [reply]
Untile Plausible releases this feature, you may take a look at Matomo or Swetrix as well, both are open-source and have `no javascript` option.

**Comment by yagarea** [comment]
Hello, is there any progress on this feature ?

**Reply by metmarkosaric** [reply]
hi @yagarea! perhaps we'll be able to support this in the future but no short term plans that i can tell you about. we're a small team so taking it one step at a time. currently we're focused on some more frequently requested features such as the comparison. as soon as we have any updates on this, we'll post in this thread. thanks for understanding!

**Comment by pateljoel** [comment]
Is there any progress on this? Simple Analytics has this feature and incorrectly assumed that Plausible also has this.

https://docs.simpleanalytics.com/without-javascript

**Comment by philipthomasme** [comment]
I'm also looking for a tracking pixel as we need it for a third party provider who does not support JavaScript tracking.

**Reply by Neuvier** [reply]
I have the same need !

**Reply by yagarea** [reply]
> I have the same need !

we all do
